wiki/HistoricalTimeline.md

```diff
@@ -1,46 +1,54 @@
 
-* 2240: All the star systems adjacent to Earth have been colonized.
+* 2230: A civil war breaks out between rival governments over colonies on Mars. The use of biological weapons encourages new rules of engagement that ban such devices in combat outside of Earth.
+
+* 2240: All the star systems adjacent to Earth have been colonized. These colonies are led by superpower nations on Earth, attempting to hold on to power and influence as corporations become interested in space exploration and dominate politics on Earth. 
 
 * 2250: Powerful governments and corporations begin exploring the galaxy with long-range scout ships and unmanned drones, "prospecting" for prime planets.
 
-* 2260: Invention of the deuterium ramscoop, allowing ships to generate their own hyperspace fuel. A privately owned colony fleet begins traveling out from Sol looking for a place to settle far from Earth, and eventually founds the colony of Valhalla.
+* 2260: A privately owned colony fleet begins traveling out from Sol looking for a place to settle far from Earth, and eventually founds the colony of Valhalla.
 
 * 2270: Colonies have developed to the point where some interstellar trade is economically feasible.
 
 * 2280: Valhalla founds a second colony in the Deep, on Midgard.
 
 * 2290: Colonies have spread to most systems within three or four jumps of Earth.
 
-* 2290: First experimental attempts at terraforming take place on Mars, with mixed results.
+* 2290: First experimental attempts at terraforming take place on Mars, with mixed results. While humans are able to live and work freely on the surface, plants and animals imported from Earth have difficulty adapting to their new conditions.
 
 ***
 
-* 2300: Most worlds are operating as individual "city-states" with their own governments, although many of those governments are in fact the corporations that funded the colonies. This is a period of relative peace, prosperity, and optimism, although starfaring ships are still far too rare and expensive to be owned by private citizens.
+* 2300: Most worlds are operating as individual "city-states" with their own governments, although many of those governments are in fact the corporations that funded the colonies or the nations that claimed the planet as their own. This is a period of relative peace, prosperity, and optimism, although starfaring ships are still far too rare and expensive to be owned by private citizens.
 
-* 2310: A small anarchist colony is established on Freedom, far north of Sol.
+* 2310: A small anarchist colony is established on Freedom, far north of Sol. The colony becomes self-sustaining, able to operate outside of the influence of any corporation or government. 
```

Comment:
```suggestion
* 2310: A small anarchist colony is established on Freedom, far north of Sol. The colony becomes self-sustaining, able to operate outside of the influence of any corporation or government. 

* 2312: In response to the new threats, Lionheart begins production of a modified colonization ship, which is intended for both commerce and convoy defense.
```

Reply:
It wouldn’t be 10 years after the pirates, for sure. 
And it’s unlikely they would do it at the same time. But, @piper8823’s choice.